hackathon_site/dashboard/frontend/src/components/general/SideSheetR/SideSheetR.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import Drawer from "@material-ui/core/Drawer";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import Button from "@material-ui/core/Button";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import LaunchIcon from "@material-ui/icons/Launch";
+import Grid from "@material-ui/core/Grid";
+
+import styles from "./SideSheetR.module.scss";
+import { ReactComponent as BackArrow } from "../../../assets/images/icons/BackArrow.svg";
+import { ReactComponent as CircleType } from "../../../assets/images/icons/redtype.svg";
+
+const createQuantityList = (number) => {
```

Comment:
This is somewhat inefficient: you have a loop to create a list of ints, which you then `.map` over below. You could just put the for loop directly inline with your `CartSheet` component below.